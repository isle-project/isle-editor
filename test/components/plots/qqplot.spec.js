// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import QQPlot from '@isle-project/components/plots/boxplot';
import heartdisease from './../fixtures/heartdisease.json';


// TESTS //

describe( '<QQPlot />', function test() {
	it( 'renders a quantile-quantile plot', () => {
		const chart = renderer.create( <QQPlot
			data={heartdisease}
			variable="Age"
		/> );
		expect( chart ).toMatchSnapshot();
	});
});
