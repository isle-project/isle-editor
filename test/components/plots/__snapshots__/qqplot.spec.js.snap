// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QQPlot /> renders a quantile-quantile plot 1`] = `
<div
  className="plotly-container"
  style={
    {
      "height": "100%",
      "width": "100%",
      "zIndex": 1,
    }
  }
>
  <button
    className="plotly-draggable-bar"
    disabled={true}
    draggable={false}
    onClick={[Function]}
    onDragStart={[Function]}
    style={
      {
        "opacity": 0.2,
      }
    }
  >
    drag-plot
  </button>
  <div
    className="plotly-draggable"
    style={
      {
        "height": "100%",
        "width": "100%",
        "zIndex": 1,
      }
    }
  />
</div>
`;
